src/apiClient.ts

```diff
@@ -42,9 +67,33 @@ class ApiClient {
     })
   }
 
-  public setTokens(tokens: Tokens): void {
-    this.tokens.accessToken = tokens.accessToken
-    this.tokens.refreshToken = tokens.refreshToken
+  public getConfiguration() {
+    return {
+      ...this.configuration,
+      configured: Object.keys(this.configuration).length > 0,
```

Comment:
what is this used for? seems like an odd thing to add to the configuration itself. if we need to perform this check somewhere (can't find where in the github diff), I'd move it there instead

Reply:
DH consumes it to check weather to re-configure the api client or not.....